src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -452,12 +440,7 @@ public synchronized Color getPixelColor(int x, int y) {
 
     /**
      * Creates an image containing pixels read from the screen.
-     * <p>
-     * If the desktop environment requires that permissions be granted
-     * to capture screen content, and the required permissions are not granted,
-     * then a {@code SecurityException} may be thrown,
-     * or the contents of the returned {@code BufferedImage} are undefined.
-     * </p>
```

Comment:
This text should not have been removed. It pertains to the desktop permissions as well as the Java SecurityManager.

Reply:
Ok, I will revert it.